Userland/Libraries/LibGUI/IconView.h

```diff
@@ -53,6 +53,9 @@ class IconView : public AbstractView {
 
     Gfx::IntSize effective_item_size() const { return m_effective_item_size; }
 
+    bool always_wrap() const { return m_always_wrap; }
+    void set_always_wrap(bool value) { m_always_wrap = value; }
```

Comment:
Let's be a little extra verbose here for clarity.. `always_wrap_item_labels` or `always_wrap_item_names`, something like that?

Reply:
I actually did fix this one up in a later commit (I think I went for `always_wrap_item_labels`), but accidentally squashed it into the main feature commit. I'll try and separate it out and squash it back into the LibGUI commit.